core/vtk/ttkUncertainDataEstimator/ttkUncertainDataEstimator.h

```diff
@@ -10,7 +10,6 @@
 /// \param Input0 Input ensemble scalar field #0 (vtkDataSet)
 /// \param Input1 Input ensemble scalar field #1 (vtkDataSet)\n
 /// ...\n
-/// \param InputN Input ensemble scalar field #N (vtkDataSet)
```

Comment:
why removing this line?

Reply:
I got this Doxygen warning:
> /home/pierre/Projects/ttk/core/vtk/ttkUncertainDataEstimator/ttkUncertainDataEstimator.h:12: warning: explicit link request to 'N' could not be resolved
This was one way of solving it.

Reply:
Another way is to prefix the `N` variable with a `%` symbol, like this:

```/// \param InputN Input ensemble scalar field #%N (vtkDataSet)```

Reply:
thanks for the trick.